adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -313,6 +315,36 @@ private[adam] class VariantContextConverter(dict: Option[SequenceDictionary] = N
   }
 
   /**
+   * Split the htsjdk variant context ID field into an array of names.
+   *
+   * @param vc htsjdk variant context
+   * @return Returns an Option wrapping an array of names split from the htsjdk
+   *    variant context ID field
+   */
+  private def splitIds(vc: HtsjdkVariantContext): Option[java.util.List[String]] = {
```

Comment:
> Can you write some unit tests for this?

As in me Justin?  I'd be happy to work on the unit tests - would be good for me.  But if @heuermh wants to as his PR he can have first dibs.   Just let me know what you prefer Michael - and if you want me to do it, should it be a separate PR or do you want me to PR against your PR Michael?